tests/api/acq_receipts/test_acq_receipts_rest.py

```diff
@@ -155,11 +155,13 @@ def test_acq_receipts_post_put_delete(client, org_martigny, vendor2_martigny,
 
 
 def test_acq_receipts_can_delete(
-        client, document, acq_receipt_fiction_martigny):
+        client, document, acq_receipt_fiction_martigny,
+        acq_receipt_line_1_fiction_martigny):
     """Test can delete an acq receipt."""
     can, reasons = acq_receipt_fiction_martigny.can_delete
-    assert can
-    assert 'links' not in reasons
+    assert not can
+    assert 'links' in reasons
```

Comment:
Maybe you can delete this, because assert reasons['links']['acq_receipt_lines'] will fail if links is missing.